sys-fs/udev/udev-230-r2.ebuild

```diff
@@ -140,6 +143,10 @@ src_prepare() {
 		echo '#define secure_getenv(x) NULL' >> config.h.in
 		sed -i -e '/error.*secure_getenv/s:.*:#define secure_getenv(x) NULL:' src/shared/missing.h || die
 	fi
+
+	use kvm && \
+		echo 'KERNEL=="kvm", NAME="%k", GROUP="kvm", MODE="0660"' \
+			> ${KVM_RULES}
```

Comment:
Yeah your likely right I did not think about the other alternatives. I guess it needs its own package then since libvirt is doing something like this already. But this is for android-studio, and other stuff likely uses /dev/kvm as well. I will make a stand alone package to create the group and stuff. Then I guess we can add use flags to that for udev, systemd, or what ever they are using. Unless it can be done universally for all but pretty sure each will need its own hooks to create device node with proper permissions.